packages/globals/test/globals.test.ts

```diff
@@ -1,8 +1,9 @@
 import { describe, it } from "mocha";
 import expect from "expect";
 import { JSDOM } from "jsdom";
+import { Symbol as SymbolOperation } from "@effection/core";
```

Comment:
The intent of the Symbol mechanism from effection was that it would be a drop-in replacement for Symbol and we would not need to alias it.

Reply:
Yeah, here alias is unnecessary